main/src/main/RunScript.scala

```diff
@@ -25,15 +25,15 @@ object RunScript{
   def runScript(home: os.Path,
                 wd: os.Path,
                 path: os.Path,
-                instantiateInterpreter: => Either[(Res.Failing, Seq[(os.Path, Long)]), ammonite.interp.Interpreter],
+                instantiateInterpreter: => Either[(Res.Failing, Seq[(() => Long, Long)]), ammonite.interp.Interpreter],
```

Comment:
Also here, some `@param` scaladoc telling what that is supposed to be would be great.